docs/content/docs/build/packages.md

```diff
@@ -1,14 +1,72 @@
 ---
 title: Packages
 menuPosition: 7
-editor: tylerbutler
-status: unwritten
+author: tylerbutler
 ---
 
-Explain what the fluid-framework package is.
-Explain the namespaces (scopes).
-List of all the packages in fluid-framework.
+The Fluid Framework is a multi-layered system consisting of dozens of individual npm packages. Most developers will only
+need two or three of these packages for typical Fluid development: The `fluid-framework` package, which contains the
+public API for the Fluid Framework, and a service-specific client package, such as the
+`@fluidframework/tinylicious-client` package.
 
-Experimental scope and packages.
+## Primary API: fluid-framework
 
-List other scopes and their meaning (or link to the wiki).
+The `fluid-framework` package bundles a collection of Fluid Framework client packages for easy use when paired with a
+corresponding service client package, such as the `@fluidframework/azure-client` package. The `fluid-framework` package
+consists primarily of two portions: the `IFluidContainer` interface and a selection of distributed data structures
+(DDSes).
+
+### IFluidContainer
+
+The [IFluidContainer][] interface is the one of
+the types returned by calls to `createContainer()` and `getContainer()` on the service clients such as `AzureClient`. It
+includes functionality to retrieve the Fluid data contained within a container, as well as to inspect the state of the
```

Comment:
how well understood is the container concept expected to be at this point?  thinking if "within the container" or "within itself" (or something else) might make more sense here.  someone unaware of the underlying container bit might be wondering about that relation

Reply:
```suggestion
includes functionality to retrieve the Fluid data contained within itself, as well as to inspect the state of the
```